src/types/games/index.ts

```diff
@@ -0,0 +1,19 @@
+import { JWTRegion } from '@/store/modules/app/authentication'
+import { Game } from '@/store/types'
+import { Ship } from './wows'
+
+export interface ApiHeader {
```

Comment:
Please note this is valid only for, wot, wows, wowp

Reply:
Maybe not, other games have APIs that we can hit

Reply:
True, however, their API can have their API specific settings. With this I mean, this should be rather called something like WargamingApiHeader.